OnlyT/Services/TalkSchedule/TalkScheduleFileBased.cs

```diff
@@ -33,9 +33,9 @@ public static IEnumerable<TalkScheduleItem> Read(bool autoBell)
 
                         foreach (XElement elem in items.Elements("item"))
                         {
```

Comment:
These changes are stylistic only and best removed so that the PR focuses on functional changes only.